packages/sdks/javascript/src/services/ApiBaseService.ts

```diff
@@ -120,6 +126,22 @@ export class ApiBaseService {
         ApiBaseService._accessToken = value;
     }
 
+    public static set onTokenExpired(callback: (accessToken: string) => void) {
+        ApiBaseService._onTokenExpired = callback;
+    }
+
+    protected isTokenExpired(response: AxiosResponse | undefined): boolean {
```

Comment:
@dudunog as all those new methods/properties might not be straightforward to all devs consuming or maintaining this API SDK, could you please add [JSDoc comments ](https://jsdoc.app/about-getting-started) to each method?

Reply:
@JeffersonAbreu Of course, but it's only one method that the user consumes to set a callback function (onTokenExpired, inside of DevopnessApiClient), right? Are there no others?

Reply:
That's fine! Adding JSDoc comments to the public methods exposed to the SDK user should be enough.